qa/os/src/test/java/org/elasticsearch/packaging/test/KeystoreManagementTests.java

```diff
@@ -253,6 +258,81 @@ public void test51WrongKeystorePasswordFromFile() throws Exception {
         }
     }
 
+    /**
+     * Check that we can mount a password-protected keystore to a docker image
+     * and provide a password via an environment variable.
+     */
+    public void test60DockerEnvironmentVariablePassword() throws Exception {
+        assumeTrue(distribution().isDocker());
+        String password = "password";
+        Path dockerKeystore = installation.config("elasticsearch.keystore");
+
+        Path localKeystoreFile = getKeystoreFileFromDockerContainer(password, dockerKeystore);
+
+        // restart ES with password and mounted keystore
+        Map<Path, Path> volumes = Map.of(localKeystoreFile, dockerKeystore);
+        Map<String, String> envVars = Map.of("KEYSTORE_PASSWORD", password);
+        runContainer(distribution(), volumes, envVars);
```

Comment:
Are we only concerned with whether ES starts up?

Reply:
Ah, what a silly oversight. I spent so long trying to get ES to start up that I forgot the line that runs our quick tests. I'll put that in.